spec/factories/organisation.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :organisation do
-    name { "MHCLG" }
+    name { "MHCLG#{SecureRandom.hex}" }
```

Comment:
Might start hitting conflicts / overlaps with some of the things I did in https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2834 (although some of the test fixes assumed we could call many orgs the same thing) - suggest merging main in ASAP to catch anything weird going on.